.github/workflows/uberjar.yml

```diff
@@ -0,0 +1,74 @@
+name: Uberjar
+
+on:
+  push:
+    branches:
+      - master
+      - 'release-**'
+    paths-ignore:
+    - 'docs/**'
```

Comment:
Can we think of some other paths to ignore?

I'm thinking about Cypress tests. Every 4th or 5th commit is a Cypress repro. I'm trying to figure out if there's any reason to build new uberjar for that (or any other test for that matter).

Btw, how does GH handle a commit that pushes both to let's say `src` and `docs/`? Is it `&&` or `||`?

Reply:
We can definitely add `frontend/tests` and `enterprise/frontend/test/` to the ignore list.

And as expected, a commit that changes both `src` and `docs` will still trigger this build.